src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -4,12 +4,15 @@ import {
   ADDRESS_POU,
   FIELD_NAMES,
   TRANSACTION_STATUS,
-} from '@@vap-svc/constants';
-import { showAddressValidationModal, inferAddressType } from '@@vap-svc/util';
-import { hasBadAddress } from '@@profile/selectors';
```

Comment:
Are we not going to be able to use the `@@profile` and `@@vap-svc` anymore?

Reply:
That's something that may have escaped the documentation (but which we will fix). Inside a workspace import paths with those aliases are not resolved by eslint, so we removed them.